solana/bridge_token_factory/programs/bridge_token_factory/src/instructions/user/finalize_transfer.rs

```diff
@@ -92,7 +85,7 @@ impl<'info> FinalizeTransfer<'info> {
         UsedNonces::use_nonce(
             data.destination_nonce,
             &self.used_nonces,
-            &mut self.config,
+            &mut self.wormhole.config,
```

Comment:
Using the config from Wormhole is a little bit confusing, can we pass it to the `post_message`?